### PR TITLE
[apiserver] fix?

### PR DESCRIPTION
**Pull request by danking**
Fix a bunch of pylint and flake things

### CODE REVIEW

File: apiserver/apiserver/apiserver.py [hunk 1]
```diff
@@ -33,6 +33,7 @@ def status_response(status):
 
 @routes.get('/healthcheck')
 async def healthcheck(request):
+    del request
```
Comment (akotlar) [inline apiserver/apiserver/apiserver.py]: What is the role of this?

File: apiserver/apiserver/apiserver.py [hunk 2]
```diff
@@ -262,25 +268,21 @@ def blocking_reference_remove_liftover(data):
...
 
 
 def blocking_parse_vcf_metadata(data):
-    return json.loads(Env.hc()._jhc.pyParseVCFMetadataJSON(data['path']))
+    return json.loads(Env.hc()._jhc.pyParseVCFMetadataJSON(data['path']))  # pylint: disable=no-member
 
 
 @routes.post('/parse-vcf-metadata')
-async def parse_vcf_metadata():
+async def parse_vcf_metadata(request):
```
Comment (akotlar) [inline apiserver/apiserver/apiserver.py]: yeah, every route needs a request, nice!

File: apiserver/apiserver/apiserver.py (hunk 1 above)
Comment (danking) [reply apiserver/apiserver/apiserver.py]: pacify pylint http://pylint-messages.wikidot.com/messages:w0613

**Comment by danking** [comment]
bump I think this will pass now

**Pull request merged**